Makefile

```diff
@@ -393,7 +393,7 @@ install-tools:
 		golang.org/x/lint/golint \
 		golang.org/x/tools/cmd/goimports \
 		github.com/securego/gosec/cmd/gosec@v0.0.0-20191008095658-28c1128b7336 \
-		sigs.k8s.io/controller-tools/cmd/controller-gen@v0.3.0 \
+		sigs.k8s.io/controller-tools/cmd/controller-gen@v0.5.0 \
```

Comment:
Did bumping this version update the CRD yamls?

Reply:
or did you run any command to generate CRD yalms?

Reply:
I ran the controller-gen command and it generates the new CRD with the version.